modules/ROOT/pages/cassandra/cassandra-get-started.adoc

```diff
@@ -0,0 +1,37 @@
+= Cassandra Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Select
+
+Cassandra Connector v1.0
+
+The Anypoint CassandraDB connector creates connectivity between Cassandra databases and third-party applications, systems, services such as SAP, PeopleSoft, or Workday as well as other databases across the enterprise. Apache Cassandra is an open source distributed NoSQL database intended to manage large amounts of product, customer, and employee data. 
+
+== Audience
+
+ * New user -- Read this document and configure your Mule application in xref:cassandra/cassandra-design-center.adoc[Design Center].   
+* Current Mule user -- Read this document and configure your Mule application in xref:cassandra/cassandra-design-center.adoc[Design Center] or xref:cassandra/cassandra-studio.adoc[Anypoint Studio]. 
+The xref:cassandra/cassandra-config-topics.adoc[Configuration Topics] help you extend your understanding of Salesforce actions.
+* Power user - Read the xref:cassandra-connector-xml-maven.adoc[XML and Maven Support], xref:cassandra/cassandra-config-topics.adoc[Configuration Topics], and xref:cassandra/cassandra-use-cases.adoc[Use Cases].
+
+== Prerequisites
+
+The Cassandra connector implements the DataStax Java Driver for Apache Cassandra 3.1. The connector exposes multiple operations in the Cassandra Connector Reference.
+
+Certain operations accept some custom POJOs as input parameters, and these are needed for the DataSense functionality.
```

Comment:
I suggest:

The following operations accept some POJOs as input parameters that are needed for the DataSense functionality:

Reply:
I don't know for sure, but trust Diane's rewrite as she seems to have a better understanding of the issue here.